### DISCUSSION TITLE
Dev tools text is massive

### DISCUSSION THREAD

**Discussion by whaaaley** [apollographql/apollo-client-devtools#1196] [Ideas]
![image](https://github.com/apollographql/apollo-client-devtools/assets/7419947/2ad48073-b3f5-414f-b245-5e66a4e7c055)

Can this be configured? Or... just normal sized?

**Comment by jerelmiller** [comment]
Hey @whaaaley 👋 

The text size adheres to our new Apollo branding with the body font at 1rem (16px) and the secondary text size set at 0.875rem (14px). Perhaps you're used to the old dev tools font size?

We don't have any plans to add a feature for adjusting text size at the moment. For now you can use the browser zoom functionality to decrease the font size if you prefer it a bit smaller.

Is there an area in particular that you're referring to here? I can certainly bring this feedback to our design team.

**Reply by whaaaley** [reply]
Thanks for the reply. The browser zoom shrinks the entire dev tools. I'd prefer if the Apollo tools fit in-line with the rest of the dev tools or system text in general. If you look at all the other text in the screenshot it's quite a bit larger, almost double. Perhaps there's some kind of zoom happening?

If you check out the Vue dev tools for comparison, everything in this screenshot is also 16px, however, it's not nearly as large as it is in the Apollo dev tools. It's much larger and it's hard to read large queries compared to how it was previously.

![image](https://github.com/apollographql/apollo-client-devtools/assets/7419947/effc33c3-0e11-42c3-9ac7-92d882fa0f48)

It's more than being used to the old dev tools. There's lots of stuff going on in dev, I typically have lots of things open at the same time. I think large text is great for a website, like when you're reading docs, presumably where the Apollo style guide comes from, but usually the default text size for code blocks in developer applications is between 12px to 14px. Fwiw, the query block is larger than what I see in the Apollo docs (14.4px).

**Reply by jerelmiller** [reply]
@whaaaley I just went back to our design file and it does indeed look like code blocks are supposed to be set at 14px. Somehow I missed that! Appreciate the callout here.

I've opened https://github.com/apollographql/apollo-client-devtools/pull/1197 to address this. Take a look at the screenshots and see if this better fits what you're looking for.

**Comment by jerelmiller** [comment]
This has been addressed with #1197 so I'll go ahead and close this 🙂
